server.go

```diff
@@ -42,6 +44,14 @@ var (
 	// ErrServerShuttingDown indicates that the server is in the process of
 	// gracefully exiting.
 	ErrServerShuttingDown = errors.New("server is shutting down")
+
+	// defaultBackoff is the starting point for exponential backoff for
+	// reconnecting to persistent peers.
+	defaultBackoff = time.Millisecond
```

Comment:
Seems too low for an initial starting value?

Reply:
Agreed, 1s might be a good starting point.

Reply:
Originally had it at 1 second but was failing itests even without perm. Swapping back now seems to be fine tho lol